src/transformers/models/gemma/modeling_gemma.py

```diff
@@ -397,7 +408,7 @@ def forward(
             query_states, key_states, value_states, attention_mask, q_len, dropout=dropout_rate
         )
 
-        attn_output = attn_output.reshape(bsz, q_len, -1).contiguous()
```

Comment:
I think this change is a regression. The shape doesn't seem to match.

Reply:
yep updating this now!